src/styles/main.scss

```diff
@@ -1,3 +1,334 @@
+@import "./utils/variables";
+@import "./utils/extends";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/main.scss";
+@import "./blocks/content.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  background: #fff;
+  box-shadow: $box-shadow;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
 body {
-  margin: 0;
+  display: flex;
+  flex-direction: column;
+  margin: $body-margin;
+  font-style: normal;
 }
+
+// $body-margin: 0;
+// $box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+// $standart-border: 1px solid #f3f3f3;
+// $blue-border: 1px solid #00acdc;
+// $nav-margin: 0;
+// $nav-padding: 0;
+// $header-padding: 0 50px;
+// $nav-link-height: 60px;
+// $nav-link-margin: 20px;
+// $nav-link-line-radius: 8px;
+// $nav-link-line-height: 4px;
```

Comment:
Don't forget to delete the comments

Reply:
Can I delete comments after final review? I left them in case of a large-scale adjustment of the task, because it is more convenient for me to work in one file in which everything is written, and not in several.